wpiutil/src/main/native/include/wpi/mDNSAnnouncer.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <string_view>
+#include <utility>
+
+#include "wpi/span.h"
+
+namespace wpi {
+class mDNSAnnouncer {
+ public:
+  mDNSAnnouncer(std::string_view serviceName, std::string_view serviceType,
+                wpi::span<std::pair<std::string, std::string>> txt);
```

Comment:
You probably want `wpi::span<const std::pair<…>>`

Reply:
You actually cant, because you can't construct a container of a const 

error C2338: The C++ Standard forbids containers of const elements because allocator<const T> is ill-formed.